src/BursarExportPlugin.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Button,
+  LoadingPane,
+  Pane,
+  PaneFooter,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+import { FormApi } from 'final-form';
+import React, { useCallback, useRef } from 'react';
+import formValuesToDto from './api/dto/to/formValuesToDto';
+import schedulingToDto from './api/dto/to/schedulingToDto';
+import useAutomaticSchedulerMutation from './api/mutators/useAutomaticSchedulerMutation';
+import useManualSchedulerMutation from './api/mutators/useManualSchedulerMutation';
+import ConfigurationForm, { FORM_ID } from './form/ConfigurationForm';
+import useInitialValues from './hooks/useInitialValues';
+import FormValues from './types/FormValues';
+import { FormattedMessage } from 'react-intl';
+
+export default function BursarExportPlugin() {
```

Comment:
I would say that default exports is not the best practice, named exports have more benefits IMO

Reply:
What benefits? IMO it's best to use default exports for situations such as this, where a file has only one primary function/responsibility

Reply:
tree shaking, IDE imports, protections while importing when people name default as they want, I believe there are more